project_one.html

```diff
@@ -0,0 +1,199 @@
+              <li class="link">css</li>
+              <li class="link">javaScript</li>
+            </ul>
+          </div>
+          <a href="#" class="pro"> See Project </a>
+        </section>
+      </div>
+
+      <br />
+      <br />
+
+      <div class="card">
+        <section class="snap">
+          <img src="./images/Snap4.png" alt="img" />
+        </section>
+        <section class="main">
+          <h1>
+            Multi-Post <br />
+            Stories
+          </h1>
+          <div class="mid">
+            <span class="can"> CANOPY </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> BACK END DEV </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> 2015 </span>
+          </div>
+          <p class="para">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="links">
+            <ul>
+              <li class="link">html</li>
+              <li class="link">css</li>
+              <li class="link">javaScript</li>
+            </ul>
+          </div>
+          <a href="#" class="pro"> See Project </a>
+        </section>
+      </div>
+      <br />
+      <br />
+
+      <section class="about">
+        <h1 class="about_h">
+          About <br />
+          Myself
+        </h1>
+        <p class="para">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <a href="#" class="conn"> LET’S CONNECT </a>
+        <div class="icons">
+          <ul>
+            <li class="icon">
+              <a href="#"> <img src="./icons/github.png" alt="img" /> </a>
+            </li>
+            <li class="icon">
+              <a href="#"> <img src="./icons/Linkedin.png" alt="img" /> </a>
+            </li>
+            <li class="icon">
+              <a href="#"> <img src="./icons/peace.png" alt="img" /> </a>
+            </li>
+            <li class="icon">
+              <a href="#"> <img src="./icons/twitter.png" alt="img" /> </a>
+            </li>
+            <li class="icon">
+              <a href="#"> <img src="./icons/m.png" alt="img" /> </a>
+            </li>
+          </ul>
+        </div>
+        <a href="#" class="pro"> Get my resume </a>
+        <div class="">
+          <ul>
+            <li>
+              Langguages <img class="iconimg" src="./icons/Union.png" alt="" />
+
+              <ul class="lang">
+                <li class="ski one"><span class="sk"> JavaScript</span></li>
+                <li class="ski two"><span class="sk"> HTML </span></li>
+                <li class="ski three"><span class="sk"> CSS </span></li>
+              </ul>
```

Comment:
- [ ] Please make sure to follow Figma design with the margin ,padding and width for each item 
![image](https://user-images.githubusercontent.com/74744787/159127756-6a9f4f8c-99e8-4730-9638-1a4f16dd47d6.png)